jobs/transforms/base_model_audits_test.py

```diff
@@ -148,3 +159,180 @@ def test_validate_post_commit_is_private_when_status_is_private(self):
         self.assert_pcoll_equal(output, [
             audit_errors.InvalidCommitStatusError(invalid_commit_status),
         ])
+
+
+class MockValidateModelDomainObjectInstancesWithNeutral(
+        base_model_audits.ValidateModelDomainObjectInstances):
+    def _get_model_domain_object_instance(self, item): # pylint: disable=unused-argument
+        return MockDomainObject()
+
+    def _get_domain_object_validation_type(self, item): # pylint: disable=unused-argument
+        return 'neutral'
```

Comment:
Please return the enum, here and elsewhere.

Reply:
Done.